sdk/devcenter/developer-devcenter-rest/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "../../../tsconfig.package",
+  "compilerOptions": {
+    "outDir": "./dist-esm",
+    "declarationDir": "./types",
+    "moduleResolution": "node"
+  },
+  "include": ["src/**/*.ts", "./test/**/*.ts"]
```

Comment:
Please update your config according the doc if you have sample: https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/RLC-quickstart.md#how-to-write-samples